__tests__/reporters/console-reporter.js

```diff
@@ -12,8 +12,10 @@ const stream = require('stream');
 // ensures consistency across environments
 require('chalk').enabled = true;
 require('chalk').supportsColor = true;
-require('chalk').styles.blue.open = '\u001b[34m';
-require('chalk').styles.bold.close = '\u001b[22m';
+// $FlowFixMe
```

Comment:
What was the error it was giving?

Reply:
property `level`. Property not found. @see https://circleci.com/gh/yarnpkg/yarn/5295?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

Just seen Yarn bundled the typed file. Not sure what's the best way to update it.

Reply:
Also,

Chalk v2 internal use `level` to determine the value:
https://github.com/chalk/chalk/blob/master/index.js#L76

and we cannot pass options into Class constructor since that return a new instance.

`supports-color` return an object and used in Chalk
https://github.com/chalk/supports-color/blob/master/index.js#L7

Both `supportsColor` & `level` is required in Chalk v2.

Reply:
This file is taken from https://github.com/flowtype/flow-typed/tree/master/definitions/npm

Bundling was most probably to reduce the compiled bundle size. Since this repo doesn't have the 2.x.x version yet, there's not much we can do. I'll add a note to the ignore comment about this.